index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Bangtan World Tour</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="wrapper">
+        <!---------------- HEADER ---------------->
+        <header class="header-content">
+            <div class="header-desktop1">
+                <nav>
+                    <ul>
+                        <li><img src="./assets/img/facebook.png" alt="facebook logo"></li>
+                        <li><img src="./assets/img/twitter.png" alt="twitter logo"></li>
+                        <li><a href="#">English</a></li>
+                        <li><a href="#">My Page</a></li>
+                        <li><a href="#">Logout</a></li>
+                    </ul>
+                </nav>
+            </div>
+            <button class="hamburger-btn"></button>  
+            <div class="header-two">
+                <img class="logo" src="./assets/img/logo.png" alt="Group logo">
+                <img class="close-btn" src="assets/img/close-icon.png" alt="">
+                <nav class="desktop-menu">
+                    <ul>
+                        <li class="nav-item"><a class="menu-item" href="./about.html">About</a></li>
+                        <li class="nav-item"><a class="menu-item" href="#Program">Program</a></li>
+                        <li class="nav-item"><a class="menu-item" href="#">Join</a></li>
+                        <li class="nav-item"><a class="menu-item" href="#">Sponsor</a></li>
+                        <li class="nav-item"><a class="menu-item" href="#">News</a></li>
+                        <li class="nav-item1"><a class="concert" href="#"></a>Spring Day World Tour</li>
+                    </ul>
+                </nav>
+            </div>
+        </header>
+        <!------------------ HOME ------------------>
+        <section class="home">
+            <div class="presentation">
+                <h1>"Hello! Armys"</h1>
+                <h2>봄날 WORLD TOUR 2025</h2>
+                <p class="text">Armys, are you ready to get hyped up ? </br>BTS are finally back to the stage to perform as a group, after a two-year hiatus. Spring Day 2025, the most awaited Wolrd Tour, is going to take them to 15 cities across Asia, Africa and Europe starting from 1st July. </br>Get your army bombs ready, many surprises are waiting for you at concert venues</p>
+                <h3>2025년7월3일 ~ 8월2일</h3>
+                <p class="venue">@SEOUL Olympic Stadium, TOKYO Dome, Prince Moulay Abdellah Stadium and more ~ </p>
+            </div>
+        </section> 
+
+        <section class="schedule">
```

Comment:
- [ ] Good job so far :clap: But I think there are a few things with your design you need to work on. Please make sure that when the topic card has hovered with the mouse you have a border around it like on the screenshot below :+1: 

- [ ] Your program card design is impressive 💯  Please make sure that the program section elements are aligned, and are on the same line, also make sure to have the same design, especially the space height, spacing padding/margin of your program card as defined on the original design :+1: 

#### Orignal design
![Template2](https://user-images.githubusercontent.com/6615394/210799135-2796a81c-2ff1-4950-9d6b-1f7687384658.jpg)